### PR TITLE
[DRAFT] FE structure checkpoint

### PR DESCRIPTION
**Pull request by alexbowen**
this got a bit tangled with a view component at one one point and we jfdi although its part of subsequent task

### CODE REVIEW

File: app/components/shared/notification_component/notification_component.js [hunk 1]
```diff
@@ -2,4 +2,4 @@
 $(document).on('click', '.js-dismissable__link', function (e) {
   e.preventDefault();
   $(this).closest('.js-dismissable').fadeOut();
-});
+});
```
Comment (cpjmcquillan) [inline app/components/shared/notification_component/notification_component.js:5]: newline

File: app/views/shared/vacancy/_jobseeker_view.html.haml [hunk 2]
```diff
@@ -31,7 +31,7 @@
   .govuk-grid-column-two-thirds
 
     - if @vacancy.organisations.many?
-      = render(NotificationComponent.new(content: t('jobs.notifications.content.multi_school_job_notification_html', organisation_type: organisation_type_basic(@vacancy.parent_organisation)), style: 'notice', background: true, dismiss: false, alert: true))
+      = render(Shared::NotificationComponent.new(content: t('jobs.notifications.content.multi_school_job_notification_html'), style: 'notice', background: true, dismiss: false, alert: true))
```
Comment (cpjmcquillan) [inline app/views/shared/vacancy/_jobseeker_view.html.haml:34]: Why has the content here been changed?

File: app/views/pages/home.html.haml [hunk 3]
```diff
@@ -4,7 +4,7 @@
   .govuk-width-container
     .govuk-grid-row
       .govuk-grid-column-two-thirds
-        .search_panel.location-search= render 'pages/home/search'
```
Comment (cpjmcquillan) [inline app/views/pages/home.html.haml:7]: There is a reason for the 2 different classes

File: app/frontend/styles/components/search_panel.scss [hunk 4]
```diff
@@ -9,7 +9,7 @@
   width: 100vw;
 }
 
-.search_panel {
```
Comment (cpjmcquillan) [inline app/frontend/styles/components/search_panel.scss:12]: As below

**Review by cpjmcquillan — APPROVED** [review]
Besides the comments, seems ok to me

File: app/views/pages/home.html.haml (hunk 3 above)
Comment (cpjmcquillan) [reply app/views/pages/home.html.haml:7]: See the subscriptions new/edit pages and the home page location inputs for comparison. If everything was under `.location-search` then the everywhere that class is applied would look like the home page search panel

File: app/frontend/styles/components/search_panel.scss (hunk 4 above)
Comment (cpjmcquillan) [reply app/frontend/styles/components/search_panel.scss:12]: Or above...

File: app/views/shared/vacancy/_jobseeker_view.html.haml (hunk 2 above)
Comment (alexbowen) [reply app/views/shared/vacancy/_jobseeker_view.html.haml:34]: good spot, im not sure am reverting this

**Pull request merged**